app/presenters/details_presenter.rb

```diff
@@ -0,0 +1,41 @@
+require "govspeak"
+
+module Presenters
+  class DetailsPresenter
+    attr_reader :content_item_details, :body
+
+    def initialize(content_item_details)
+      @content_item_details = SymbolizeJSON.symbolize(content_item_details)
+      @body = content_item_details[:body]
+      @body = [@body] if @body.is_a?(Hash)
```

Comment:
Yes! I didn't know that existed, I've been getting caught out running `Array()` on hashes, it looks like that resolves just that. 

@dougdroper you might be stoked if you didn't know about this before - or maybe you know a downside I don't know?